docs/v1beta1-checklist-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+---
+title: v1beta1 Checklist
+toc: true
+weight: 740
+indent: true
+---
+# v1beta1 Checklist
+
+Crossplane adheres to the [Kubernetes definition] of API versioning. This means
+that when an API type reaches a `beta` level of quality it should be
+well-tested, enabled by default, and will be supported through subsequent
+releases, even if the details of the type change. As a community, we implement
+`beta` API types when the following are true:
+
+- An `alpha` version of the API types currently exits.
+- We are committed to long-term support the API type.
+- There is demonstrated interest for support of the API type in the end-user
+  community.
+
+When the above are all true, the first version of a `beta` level API type,
+`v1beta1`, should be implemented in accordance with the specifications described
+in the remainder of this document.
+
+## Areas of Quality
+
+Crossplane controllers are comprised of four main components: the cloud provider
+API type, the Crossplane API type, the client, and the controller. Each of these
+components serves a specific purpose that should not overlap with any of the
+other components. It is helpful to conceptualize these components in the
+following manner:
+
+- **Cloud Provider API Type**: the language the controller speaks to manage the
```

Comment:
I think `language` is kind of confusing here, maybe `struct` or `object` which is what it really is in the end?

Reply:
@muvaf in the section directly below this the terms are described in a more concrete manner. this section is meant to create a conceptual model, and the next is meant to describe implementation. If you do not favor this structure, I would recommend removal of the conceptual section because refactoring as suggested will essentially make it a copy of the section directly below.